package.json

```diff
@@ -28,7 +28,7 @@
   "scripts": {
     "build-update-h3": "bash scripts/update-h3.sh",
     "build-emscripten": "yarn build-update-h3 && yarn docker-emscripten",
-    "build-docs": "jsdoc2md --global-index-format grouped --partial doc-files/scope.hbs --helper ./doc-files/insert-version.js --separators --template doc-files/README.tmpl.md lib/h3core.js > README.md",
+    "build-docs": "jsdoc2md --no-cache --global-index-format grouped --partial doc-files/scope.hbs --helper ./doc-files/insert-version.js --separators --template doc-files/README.tmpl.md lib/h3core.js > README.md",
```

Comment:
Pretty sure that this + the JSDoc bump should fix problems in #91 - cc @isaacbrodsky